pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_creating-a-product.adoc

```diff
@@ -0,0 +1,28 @@
+[id="creating-a-product_{context}"]
+= Creating a product
+
+A product must exist in {ProductName} before you can assign a product name to the metadata of a module or assembly. If the product doesn't already exist, you need to create one.
+
+.Prerequisites
+
+* You are logged in.
+* You have Publisher permissions.
```

Comment:
Should we mark up "Publisher"? I'm honestly not sure.

Reply:
Well, in RHV documenation, I see that the roles are in bold.